doc/quickstart/geostyler_quickstart.rst

```diff
@@ -0,0 +1,132 @@
+:Author: Jan Suleiman
+:Version: osgeolive14.0draft
+:License: Creative Commons Attribution-ShareAlike 3.0 Unported  (CC BY-SA 3.0)
+
+@LOGO_geostyler@
+@OSGEO_KIND_geostyler@
+
+********************************************************************************
+@NAME_geostyler@ Quickstart
+********************************************************************************
+
+This Quickstart describes some basic steps required to start working with GeoStyler: installing GeoStyler and adding a first UI component to your react application.
+
+GeoStyler makes it easy to style geodata in any web page. The UI is independent from any styling format and should be usable for e.g. OpenLayer or SLD.
+
+.. contents:: Contents
+    :local:
+
+Installing GeoStyler
+====================
+
```

Comment:
The installation was done on OSGeoLive already. You could skip that part or add a sentence how to access the application on OSGeoLive.

Maybe make it more concrete. What is your project root? Please give a defined example. 

Keep in mind that the OSGeoLive user may be absolute beginners.

Reply:
done in https://github.com/OSGeo/OSGeoLive-doc/pull/604/commits/79b892a57dc759bc650410399b965dbbf2bcdd69